### PR TITLE
Changed style line and added a button

### PR DESCRIPTION
**Pull request by tech-anand247**
nothing to describe

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1 +1 @@
-new text here.
\ No newline at end of file
+first line
```
Comment (getfullstack) [inline styles.css]: Please change it to second.

**Review by getfullstack — CHANGES_REQUESTED** [review]

File: index.htm [hunk 2]
```diff
@@ -0,0 +1 @@
+<button>click me</button>
```
Comment (getfullstack) [inline index.htm]: Make them capital .

Comment (getfullstack) [inline index.htm]: add another button with small letters

**Review by getfullstack — CHANGES_REQUESTED** [review]

File: styles.css (hunk 1 above)
Comment (tech-anand247) [reply styles.css]: Sure. Thanks.

File: index.htm (hunk 2 above)
Comment (tech-anand247) [reply index.htm]: ok

Comment (tech-anand247) [reply index.htm]: can you please confirm with team

**Review by tech-anand247 — COMMENTED** [review]
Please review and merge.

**Review by getfullstack — APPROVED** [review]

File: index.htm (hunk 2 above)
Comment (getfullstack) [reply index.htm]: add another

**Review by getfullstack — APPROVED** [review]

**Pull request merged**